modules/ROOT/pages/release-notes.adoc

```diff
@@ -23,6 +23,14 @@ These credentials can be consulted and modified:
 * in the <BONITA>/server/conf/tomcat-users.xml file in a *Bonita Tomcat bundle*
 * using xref:bonita-docker-installation.adoc#_monitoring_username[specific Environment parameters] when starting a *Bonita Docker image*
 
+
+=== Dynamic authorization checking active by default (Subscription editions)
+
+xref:identity:rest-api-authorization.adoc#dynamic_authorization[Dynamic authorization checking] is now only available in the *Enterprise* editions, and is now *active by default* for new installations. +
+Existing installations that update to 2022.1 stay *unchanged*, but we encourage customers to activate dynamic authorization checking, once they have benched that the performance cost is reasonable.
+
+Community users that update to 2022.1 simply lose the possibility to use dynamic authorization checking, and their potential previous configuration regarding dynamic authorizations is automatically cleared out from the installation, during the update procedure.
```

Comment:
Looks like a huge breaking change, I think it should be highlighted correctly to ensure that everybody get the message..

Reply:
Finally, we changed that behaviour.
I updated the doc accordingly.
Thx for the feedback @alachambre